course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+
+### Week 1
+1. Introduction to the course, intro to the ecology part, motivation behind more reproducible workflows, talk briefly about spreadsheet and get straight into how to use R.
+    - Since Martin mentioned we should expect no programming experience from these students, I want to introduce programming at the first lecture already, at least conceptually.
+    - We can do this based on a [jupyter notebook I wrote previously](https://github.com/joelostblom/resources/blob/master/from-spreadsheets-to-pandas/from-spreadsheets-to-pandas.ipynb) with analogies between spreadsheet software and programming languages. It's in Python, but if you think this is a good idea, we can adopt it for R. If not, we can do something else.
+2. Basic R programming, vectors, data frames, loops, conditions, etc.
+3. Before 2nd class: Everyone have Rstudio up and running. Maybe we can also link them to some R intro course and ask them to complete 1 or 2 lectures there per week, which will serve as the underlying material for the quiz? This one is good imo http://tryr.codeschool.com/. Watch Christie's talk https://www.youtube.com/watch?v=TthxI9mlvGE?
+
+### Week 2
+1. Basic R-programming, repetition and continued, scripting.
+2. dplyr intro, finalize R-prep by teaching outstanding items required to use R for the following ecology classes.
+3. Again, link online lectures?
+
+### Week 3
+1. Ecology background knowledge for lab 1
+2. Ecology lab 1 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 4
+1. Ecology background knowledge for lab 2
+2. Ecology lab 2 (or similar)
+3. Lab report? Aka send in their code.
+
+### Week 5
+1. Introduce data sets, divide intro groups, think about interesting questions to ask, formulate hypothesis/plan of action.
+    - Potentially do this already the week before and have them take a survey with the data they want to work on and their background experience.
+2. Project work, getting started with their own data in R, tidy data principles, how to clean data, setting up a collaborative git environment with Rstudio (how much do we want to get into git?).
+3. Reflections blog post of what they have learnt so far?
+
+### Week 6
+1. Data visualization in general and in R, ggplot, interactive (since plotly is just one additional line)?
+2. Data visualization continued, project specific.
+3. Online tutorial about plotting.
+
+### Week 7
+1. Stuff important for reproducible workflow, metadata, Rmarkdown, Roxygen?
+2. Project work,
+3. Write a data README?
+
+### Week 8-12
+I have left these open for now, since I expect there will be many things that will be reorganized anyways. Ideally one lecture per week from now on would be project work only. Potential ideas:
+    - A couple of additional ecology lessons?
+    - More info about working together on online collaborative platforms? Both for code and writing.
+    - More about data openness?
+    - Guest lectures?
+    - Preparing a scientific item for academic publishing
```

Comment:
I think this lesson is important/necessary as we expect some final project output (a report, or manuscript or poster).